1674112884564	Sync.LogManager	DEBUG	Flushing file log
1674112884572	Sync.LogManager	DEBUG	Log cleanup threshold time: 1673248884572
1674112884579	Sync.LogManager	DEBUG	Done deleting files.
1674112886259	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1674112891379	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1674112891379	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1674112891381	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1674112891382	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1674112900569	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1674112900569	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1674112900571	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1674112900571	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1674112900572	Sync.Service	DEBUG	User-Agent: Firefox/108.0.2 (Linux x86_64) FxSync/1.110.0.20230104165113.desktop
1674112900572	Sync.Service	INFO	Starting sync at 2023-01-19 12:51:40 in browser session EWOqlYVHJhTm
1674112900572	Sync.Service	DEBUG	In sync: should login.
1674112900572	Sync.Service	INFO	User logged in successfully - verifying login.
1674112900572	Sync.SyncAuthManager	DEBUG	unlockAndVerifyAuthState has no FxA user
1674112900572	Sync.Status	DEBUG	Status.login: error.login.reason.no_username => error.login.reason.no_username
1674112900572	Sync.Status	DEBUG	Status.service: service.client_not_configured => service.client_not_configured
1674112900572	Sync.Service	DEBUG	Fetching unlocked auth state returned error.login.reason.no_username
1674112900572	Sync.ErrorHandler	ERROR	Sync encountered a login error
1674112900572	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1674112900572	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1674112900572	Sync.Service	DEBUG	Exception calling WrappedLock: Error: Login failed: error.login.reason.no_username(resource://services-sync/service.js:1041:15) JS Stack trace: onNotify@service.js:1041:15
1674112900572	Sync.Service	DEBUG	Not syncing: login returned false.
